core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/OrderByClause.kt

```diff
@@ -2,49 +2,47 @@ package ch.ergon.dope.resolvable.clause.model
 
 import ch.ergon.dope.DopeQuery
 import ch.ergon.dope.DopeQueryManager
+import ch.ergon.dope.resolvable.Resolvable
 import ch.ergon.dope.resolvable.clause.ISelectGroupByClause
 import ch.ergon.dope.resolvable.clause.ISelectOrderByClause
-import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.expression.TypeExpression
 import ch.ergon.dope.resolvable.formatToQueryStringWithSymbol
-import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-enum class OrderByType(val type: String) {
+enum class OrderByType(val queryString: String) {
     ASC("ASC"),
     DESC("DESC"),
 }
 
 private const val ORDER_BY = "ORDER BY"
 
-open class SelectOrderByClause<T : ValidType>(private val stringField: Field<StringType>, private val parentClause: ISelectGroupByClause<T>) :
-    ISelectOrderByClause<T> {
-
+class OrderExpression(private val typeExpression: TypeExpression<out ValidType>, private val orderByType: OrderByType? = null) : Resolvable {
```

Comment:
It's possible to use aggregate functions in order by clauses, currently that's not possible because they are Unaliased Expressions. I think it's not in the scope of this ticket though, do you think we should create a new ticket to support that?

Reply:
I created a ticket: DOPE-268